### PR TITLE
Revamp the Overview article

### PR DESCRIPTION
**Pull request by anastasiia-fedotova**
restructured
added Server settings and Security settings
screenshots will be updated when version 2.5 is released

### CODE REVIEW

File: docs/private-dns/overview.md [hunk 1]
```diff
@@ -13,21 +13,21 @@ Quick link: [Try AdGuard DNS](https://agrd.io/download-dns)
 
 ![Private AdGuard DNS dashboard main](https://cdn.adtidy.org/public/Adguard/Blog/private_adguard_dns/main.png)
 
-## What is Private AdGuard DNS?
+## General
 
 <iframe width="560" height="315" class="youtube-video" src="https://www.youtube-nocookie.com/embed/ME3_Ms9LO8M" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
-In addition to the benefits of a public DNS server (such as traffic encryption and domain blocklists), Private AdGuard DNS provides features like flexible customization, DNS statistics, and parental control, and is easily managed through a handy dashboard.
+Private AdGuard DNS offers all the advantages of a public AdGuard DNS server, including traffic encryption and domain blocklists. Moreover, it provides additional features such as flexible customization, DNS statistics, and parental control. All these capabilities are easily accessible and managed via a user-friendly dashboard.
```
Comment (el-termikael) [inline docs/private-dns/overview.md:20]:
```suggestion
Private AdGuard DNS offers all the advantages of a public AdGuard DNS server, including traffic encryption and domain blocklists. It also offers additional features such as flexible customization, DNS statistics, and Parental control. All these options are easily accessible and managed via a user-friendly dashboard.
```

File: docs/private-dns/overview.md [hunk 2]
```diff
@@ -13,21 +13,21 @@ Quick link: [Try AdGuard DNS](https://agrd.io/download-dns)
...
+At one time, the AdGuard product line included only [public AdGuard DNS](../public-dns/overview.md) and [AdGuard Home](https://github.com/AdguardTeam/AdGuardHome). These solutions work fine for some users, but for others, the public AdGuard DNS lacks the flexibility of configuration, while the AdGuard Home lacks simplicity. That's where private AdGuard DNS comes into play. It has the best of both worlds: it offers customizability, control and information - all through a simple easy-to-use dashboard.
 
-## The difference between Private and Public AdGuard DNS
+### The difference between private and public AdGuard DNS
 
-Here is a simple comparison of features available in Public AdGuard DNS and Private AdGuard DNS servers.
+Here is a simple comparison of features available in public AdGuard DNS and private AdGuard DNS servers.
```
Comment (el-termikael) [inline docs/private-dns/overview.md:30]:
```suggestion
Here is a simple comparison of features available in public and private AdGuard DNS.
```

File: docs/private-dns/overview.md [hunk 3]
```diff
@@ -13,21 +13,21 @@ Quick link: [Try AdGuard DNS](https://agrd.io/download-dns)
...
 
-We already have [public AdGuard DNS](../public-dns/overview.md) and [AdGuard Home](https://github.com/AdguardTeam/AdGuardHome). These solutions work fine for some users, but for others, the public AdGuard DNS lacks the flexibility of configuration, while the AdGuard Home lacks simplicity. That's where private AdGuard DNS comes into play. It has the best of both worlds: it offers customizability, control and information - all through a simple easy-to-use dashboard.
+At one time, the AdGuard product line included only [public AdGuard DNS](../public-dns/overview.md) and [AdGuard Home](https://github.com/AdguardTeam/AdGuardHome). These solutions work fine for some users, but for others, the public AdGuard DNS lacks the flexibility of configuration, while the AdGuard Home lacks simplicity. That's where private AdGuard DNS comes into play. It has the best of both worlds: it offers customizability, control and information - all through a simple easy-to-use dashboard.
 
-## The difference between Private and Public AdGuard DNS
+### The difference between private and public AdGuard DNS
```
Comment (el-termikael) [inline docs/private-dns/overview.md:28]:
```suggestion
### The difference between public and private AdGuard DNS
```

File: docs/private-dns/overview.md [hunk 4]
```diff
@@ -110,10 +100,30 @@ This is a detailed log where you can check out the information on every single r
 
 ![Private AdGuard DNS dashboard query log](https://cdn.adtidy.org/public/Adguard/Blog/private_adguard_dns/query_log.png)
 
+## Server settings
+
+This section features a range of settings allowing you to customize the operation of private AdGuard DNS, ensuring the Internet functions exactly as you desire.
+
+### Blocklists management
+
+With *Blocklists* feature you can set which domains you want to block and which you don't. Сhoose from wide variety of blocklists for different purposes.
```
Comment (el-termikael) [inline docs/private-dns/overview.md:109]:
```suggestion
The *Blocklists* feature allows you to specify which domains you want to block and which you don't. Choose from a variety of blocklists for different purposes.
```

File: docs/private-dns/overview.md [hunk 5]
```diff
@@ -110,10 +100,30 @@ This is a detailed log where you can check out the information on every single r
...
+
+![Private AdGuard DNS dashboard blocklists](https://cdn.adtidy.org/public/Adguard/Blog/private_adguard_dns/blocklists.png)
+
+### Security settings
+
+Even if you are aware of all the tricks of online scammers, there's always a risk of accidentally clicking a harmful link. To protect yourself from such accidents, go to the *Security settings* section and check the boxes next to the options listed there.
```
Comment (el-termikael) [inline docs/private-dns/overview.md:115]:
```suggestion
Even if you're aware of all the tricks online scammers use, there's always a chance you'll accidentally click a malicious link. To protect yourself from such accidents, go to the *Security settings* section and check the boxes next to the options listed there.
```

File: docs/private-dns/overview.md [hunk 6]
```diff
@@ -110,10 +100,30 @@ This is a detailed log where you can check out the information on every single r
...
+
+The *Block malicious, phishing, and scam domains* feature will block domains found in the dedicated database. And the *Block newly registered domains* will block all domains registered less than 30 days ago, which are often considered risky for your online privacy.
+
+### User rules
+
+For times when pre-installed *Blocklists* with thousands of rules are not enough, we have a handy function called *User rules*. Here you can add custom rules manually to block/unblock a certain domain or import custom rules lists (check out [DNS filtering rules syntax](../general/dns-filtering-syntax.md)). You can export the lists.
```
Comment (el-termikael) [inline docs/private-dns/overview.md:121]:
```suggestion
For cases where pre-installed blocklists with thousands of rules are not enough, we have a handy feature called *User rules*. Here you can manually add custom rules to block/unblock a specific domain or import custom rule lists (see [DNS filtering rules syntax](../general/dns-filtering-syntax.md)). You can export the lists.
```

**Review by el-termikael — CHANGES_REQUESTED** [review]

**Review by el-termikael — APPROVED** [review]

File: docs/private-dns/overview.md [hunk 7]
```diff
@@ -110,10 +100,30 @@ This is a detailed log where you can check out the information on every single r
...
+
+### Blocklists management
+
+The *Blocklists* feature allows you to specify which domains you want to block and which you don't. Choose from a variety of blocklists for different purposes.
+
+![Private AdGuard DNS dashboard blocklists](https://cdn.adtidy.org/public/Adguard/Blog/private_adguard_dns/blocklists.png)
+
+### Security settings
+
+Even if you're aware of all the tricks online scammers use, there's always a chance you'll accidentally click a malicious link. To protect yourself from such accidents, go to the *Security settings* section and check the boxes next to the options listed there.
```
Comment (ameshkov) [inline docs/private-dns/overview.md:115]: Screenshots?

File: docs/private-dns/overview.md [hunk 8]
```diff
@@ -58,7 +60,7 @@ Here is a simple comparison of features available in Public AdGuard DNS and Priv
 
 Every device that you add in the AdGuard DNS panel has its own unique address that can be used if the device supports modern encrypted DNS protocols (DoH, DoT, and DoQ).
 
-## Linked IP
+### For devices that do not support DoH, DoT, and DoQ
```
Comment (ameshkov) [inline docs/private-dns/overview.md:63]: Linking IP is not the only way, there's also dedicated IPv6 addresses which is a better option (but not everyone can use it).

**Review by ameshkov — CHANGES_REQUESTED** [review]

File: docs/private-dns/overview.md (hunk 7 above)
Comment (anastasiia-fedotova) [reply docs/private-dns/overview.md:115]: i'll add them later along with updating other screenshots of this article after v2.5 release -- if you don't mind

File: docs/private-dns/overview.md [hunk 9]
```diff
@@ -62,7 +62,25 @@ Every device that you add in the AdGuard DNS panel has its own unique address th
 
 ### For devices that do not support DoH, DoT, and DoQ
 
-If the device does not support encrypted DNS and you have to use plain DNS, there's another way to allow AdGuard DNS to recognize the device — link its IP address. In this case AdGuard DNS counts all plain DNS requests that come from that IP address towards that "device".
+If the device does not support encrypted DNS and you have to use plain DNS, there are two more ways to allow AdGuard DNS to recognize the device — use dedicated IP addresses or link device's IP address.
+
+:::note
+
+Use plain DNS addresses only if you have no other options: this reduces the security of DNS requests. If you decide to use plain DNS, we recommend choosing dedicated IP addresses.
```
Comment (el-termikael) [inline docs/private-dns/overview.md:69]:
```suggestion
Use plain DNS addresses only if you have no other options: this reduces the security of DNS requests. If you decide to use plain DNS, we recommend that you choose dedicated IP addresses.
```

File: docs/private-dns/overview.md [hunk 10]
```diff
@@ -62,7 +62,25 @@ Every device that you add in the AdGuard DNS panel has its own unique address th
...
+
+:::
+
+#### Dedicated IP addresses
+
+For every device that you connect to AdGuard DNS, you'll be offered two dedicated IPv6 addresses that you can enter in your device settings. Using both IPv6 addresses is not mandatory, but often devices might request you to enter two IPv6 addresses.
+
+When you connect to them, AdGuard DNS will be able to determine which particular device is sending DNS requests and display statistics for it. And you'll be able to configure DNS rules specifically for this device.
+
+Unfortunately, not all service providers offer IPv6 support, and not all devices allow you to configure IPv6 addresses. If it is your case, you might need to rely on the Linked IP method.
```
Comment (el-termikael) [inline docs/private-dns/overview.md:79]:
```suggestion
Unfortunately, not all service providers offer IPv6 support, and not all devices allow you to configure IPv6 addresses. If this is your case, you may have to rely on the Linked IP method.
```

File: docs/private-dns/overview.md [hunk 11]
```diff
@@ -62,7 +62,25 @@ Every device that you add in the AdGuard DNS panel has its own unique address th
...
+
+#### Linked IP
+
+If you connect your device to AdGuard DNS through Linked IP the service counts all plain DNS requests that come from that IP address towards that "device". With this connection method, you would have to reconnect manually or via a special program every time the device's IP changed, which occurred after every reboot.
```
Comment (el-termikael) [inline docs/private-dns/overview.md:83]:
```suggestion
If you connect your device to AdGuard DNS via Linked IP, the service will count all plain DNS requests coming from that IP address towards that "device". With this connection method, you would have to reconnect manually or through a special program each time the device's IP changes, which happens after each reboot.
```

**Review by el-termikael — CHANGES_REQUESTED** [review]

**Review by el-termikael — APPROVED** [review]

**Pull request merged**